content/docs/addons-test-utils.md

```diff
@@ -322,19 +322,19 @@ Simulate.{eventName}(
 )
 ```
 
-Simulate an event dispatch on a DOM node with optional `eventData` event data.
+رخداد یک رویداد روی یک نود DOM را با داده رویداد اختیاری `eventData` شبیه‌سازی می‌کند.
```

Comment:
```suggestion
توضیع یک رویداد روی یک نود DOM با `eventData` داده اختیاری برای رویداد شبیه‌سازی می‌کند.
```

Reply:
Some typo in your suggestion. So i will commit correct edit.,

Reply:
Good catch!